portfolio/src/main/webapp/index.html

```diff
@@ -98,10 +96,17 @@ <h3 id="contact_me">Contact Me</h3>
       <a href="https://www.github.com/KathyFeiyang">
         <img src="/images/GitHub_icon.png" width="50" height="50">
       </a>
-      <p><b>Submit feedback :)</b></p>
-      <form id="feedback-form" onsubmit="presentFeedbackReceipt()" action="/index.html#contact_me">
+      <p><b>Comment History</b></p>
+      <label for="maxN">Maximum number of entries to show in comment history:</label>
+      <input type="number" id="maxN" name="maxN" onchange="addComments()">
```

Comment:
Maybe instead of validating in the servlet, you can just specify limits to the textbox itself: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

Reply:
Ok! Then I can query for the number of existing comments in the database

Reply:
I added the limits using HTML. I tested this and discovered that the min and max limits only apply to the range of values you can choose using the up & down arrow icons. Users can still manually type in an invalid value in the textbox if they desire. Because of this, I kept the validation code in the Java servlet.

Reply:
The max limit is updated to reflect the actual total number of comments in the comment history.